packages/frontend-api/apiary.apib

```diff
@@ -1295,6 +1295,30 @@ Login user and return access and refresh tokens
         }
     }
 
+## Logout [/graphql]
+
+### POST [POST]
+
+Logout user
+
+- Request (application/json)
+
+    - Headers
+        Authorization: Bearer ABCDEF
```

Comment:
I would use the token from the previous example so it is more clear :)

Reply:
I dont't think so, see below

Reply:
b46de52fdd97b7c73b5d49e25245679a7b3e8886